torch_glow/tests/nodes/quantized_linear_test.py

```diff
@@ -5,6 +5,56 @@
 from tests.utils import jitVsGlow
 
 
+def test_quantized_linear_unpacked():
+    """Basic test of the PyTorch quantized::linear_unpacked Node on Glow."""
+
+    def test_f(inputs, weights, bias=None):
+        q_int = torch.nn.quantized.Quantize(
+            scale=1/15, zero_point=0, dtype=torch.qint8)
+        q_uint = torch.nn.quantized.Quantize(
+            scale=1/10, zero_point=15, dtype=torch.quint8)
+
+        dq = torch.nn.quantized.DeQuantize()
+
+        q_inputs = q_uint(inputs)
+        q_weights = q_int(weights)
+
+        return dq(torch.nn.quantized.functional.linear(q_inputs, q_weights, bias))
+
+    inputs = torch.tensor([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]])
+    weights = torch.tensor([[2.0, 4.0], [3.0, 1.0], [6.0, 1.0]])
+    bias = torch.tensor([5.0, 3.0, 2.0])
+
+    jitVsGlow(test_f, inputs, weights, bias, expected_fused_ops={
+              "glow::unpacked_quantized_linear", "aten::quantize_per_tensor",
+              "aten::dequantize"})
+
+
+def test_quantized_linear_packed():
+    """Basic test of the PyTorch quantized::linear Node on Glow."""
+
+    q = torch.nn.quantized.Quantize(scale=1/25, zero_point=17,
+                                    dtype=torch.quint8)
+    dq = torch.nn.quantized.DeQuantize()
+    linear = torch.nn.Linear(5, 5)
+
+    linear.weight.data.fill_(1.2)
+    linear.bias.data.fill_(3.0)
+
+    model = torch.nn.Sequential(q, linear, dq)
+    model.qconfig = torch.quantization.get_default_qconfig('fbgemm')
+    torch.quantization.prepare(model, inplace=True)
+    torch.quantization.convert(model, inplace=True)
+
+    x = torch.tensor(range(5), dtype=torch.float)
```

Comment:
Unpacked linear had enough accuracy to use random inputs, does this not?

Reply:
Yes we already have a random unit test under this. Think I can delete this.